string object_class
float64 probability
geometry_msgs/Point ul_point
geometry_msgs/Point br_point
geometry_msgs/Point size_px
